build/yaml/js-build-steps.yml

```diff
@@ -91,6 +91,13 @@ steps:
     verbose: false
     customCommand: 'run test:devops'
 
+- task: Npm@1
+  displayName: 'npm run test:consumer'
+  inputs:
+    command: custom
+    verbose: false
+    customCommand: 'run test:consumer'
+
```

Comment:
Not necessary to address in this PR, but one thing to consider for the future is placing neatly isolated tests like this in a GitHub Workflow.

Reply:
We can definitely look in to that, I don't mind using a consistent CI platform though. What are the benefits to GH Actions vs. Azure DevOps?

Reply:
TBH I've only performed a 1-dimensional analysis on using Workflows vs DevOps...

My overarching CI goals are are:
- keep DevOps CI duration the same or reduce the duration with any CI-related PR
- do not sacrifice code quality for DevOps CI duration
- keep Agents free for parallel builds (a sub-point of the first one)

This PR would increase duration for the existing DevOps pipelines, and the task doesn't output any artifacts for publishing which is why putting it in a Workflow sounds appealing to me. (We could also produce artifacts from Workflow, so the second point is slightly moot).

One initial reason as to why I wouldn't put this particular task in a Workflow at the moment:
The test installs the latest of 9 different `Major.minor` versions of typescript. In Azure DevOps it's relatively trivial to setup a task/stage that utilizes npm or DevOps caching mechanisms to improve the speed of the task. This still doesn't reduce the amount of agents being used, but there might be a great enough speed boost to warrant keeping it in DevOps.

That said, it _shouldn't_ be difficult to setup a container or setup with the cached versions of Typescript for use in a Workflow; I've just never done it before 😄